.buildkite/pipeline.yml

```
@@ -26,8 +29,11 @@ steps:
     agents:
       provider: gcp
     plugins:
-      - *google_oidc_plugin:
-          use-service-account: true
+      - elastic/oblt-google-auth#${BUILDKITE_COMMIT}:
+        lifetime: 10800 # seconds
+        project-id: "elastic-observability-ci"
+        project-number: "911195782929"
+        use-service-account: true
```

```suggestion
          lifetime: 10800 # seconds
          project-id: "elastic-observability-ci"
          project-number: "911195782929"
          use-service-account: true
```

See https://buildkite.com/docs/pipelines/integrations/plugins/using#adding-a-plugin-to-your-pipeline